### PR TITLE
Moving some utility classes to library for connector reuse

### PR DESCRIPTION
**Pull request by vyshane**
These util classes have been moved/adapted from wisetime-jira-connector.

Have also fixed some issues that @sukolenvo found/suggested in [this PR](https://github.com/wisetime-io/wisetime-jira-connector/pull/3/files/7a2fda8ffe156cf4ece34c12eca2e6bf826ad59e#diff-b8c79ab7191a23eb86a4a5e685c58584).

Next we'll update the wisetime-jira-connector project.

### CODE REVIEW

**Comment by vyshane** [comment]
FYI @alvinllobrera

File: src/main/java/io/wisetime/connector/testutils/FakeEntities.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        .groupName(FAKER.color().name())
+        .tags(randomEntities(() -> randomTag(TAG_PATH), 1, 3))
+        .user(randomUser())
+        .timeRows(timeRows)
+        .narrativeType(randomEnum(TimeGroup.NarrativeTypeEnum.class))
+        .durationSplitStrategy(randomEnum(TimeGroup.DurationSplitStrategyEnum.class));
+  }
+
+  public Tag randomTag(final String path) {
+    return new Tag()
```
Comment (sukolenvo) [inline src/main/java/io/wisetime/connector/testutils/FakeEntities.java]: Unused variable `path`
```suggestion
    return new Tag()
        .path(path)
```

File: src/main/java/io/wisetime/connector/testutils/FakeEntities.java [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  }
+
+  private <T> List<T> randomEntities(final Supplier<T> supplier, final int min, final int max) {
+    return IntStream
+        .range(0, FAKER.random().nextInt(min, max))
+        .mapToObj(i -> supplier.get())
+        .collect(Collectors.toList());
+  }
+
+  private static <T extends Enum<?>> T randomEnum(Class<T> clazz) {
```
Comment (sukolenvo) [inline src/main/java/io/wisetime/connector/testutils/FakeEntities.java]:
```suggestion
  private static <T extends Enum<?>> T randomEnum(final Class<T> clazz) {
```
Missing `final` modifier

**Comment by vyshane** [comment]
Thanks for suggestions, have applied

**Pull request merged**